bot/cogs/fun.py

```diff
@@ -175,6 +180,14 @@ def __init__(self, bot: Bot):
         embed_quotation = quotation.embeds[0]
         await ctx.send(embed=embed_quotation)
 
+    @command()
```

Comment:
This needs premissions checking otherwise it shall immediately be broken.

Reply:
It has been added. See the line below.

Reply:
I can't see a permissions check?

Reply:
@has_any_role(*ADMIN_ROLES)